packages/frontend/src/components/tags.tsx

```diff
@@ -2,21 +2,26 @@ import { Sport } from "@esp-group-one/types";
 
 interface Tagging {
   sportName: Sport;
+  active: boolean;
 }
 
-export function Tag({ sportName }: Tagging) {
+export function Tag({ sportName, active }: Tagging) {
   return (
     <div
       className={`inline-block rounded-full py-1 px-5 font-bold font-title text-white text-md ${getSportColorClass(
         sportName,
+        active,
```

Comment:
Inline with the above recommendation
```suggestion
        active ?? true,
```